core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -630,5 +632,20 @@ class CityInfo {
         return !isOriginalCapital && !isCapital() && !isHolyCity()
     }
 
+
+    fun getNeighbouringCivs(): List<String> {
+        val tilesList: ArrayList<TileInfo> = getTiles().toList() as ArrayList<TileInfo>
```

Comment:
Maybe this would be better as a hashset, since you're mostly checking if it contains a certain value

Reply:
yeah the code looks nicer. thanks for the review